week-4/E-arrays-of-objects/exercise-3.js

```diff
@@ -54,32 +54,52 @@ DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
 
-
 var restaurantFinderApplication = {
-    applicationName: "Restaurant Finder",
-    applicationVersion: "1.0",
-    restaurants: restaurants,
-    findAvailableRestaurants: function(numberOfPeople) {
-        // Complete here
-    },
-    findRestaurantServingDish: function(dishName) {
-        // Complete here
-    },
-    countNumberOfRestaurantsInArea: function(area) {
-        // Complete here
-    }
+  applicationName: "Restaurant Finder",
+  applicationVersion: "1.0",
+  restaurants: restaurants,
+  findAvailableRestaurants: function(numberOfPeople) {
+    var availableRestaurants = restaurants.filter(function(restaurant) {
+      return;
+      restaurant.totalSeats - restaurant.numberOfCustomers >= numberOfPeople;
+    });
+
+    return availableRestaurants.map(function(restaurant) {
+      return restaurant.name;
+    });
+  },
+  findRestaurantServingDish: function(dishName) {
```

Comment:
The next two functions need an implementation. Could you let me know once you completed them?